### PR TITLE
[Thor] Update to v0.19.4

### PR DESCRIPTION
**Pull request by segiddins**
Closes #4876

@rafaelfranca

### CODE REVIEW

**Comment by segiddins** [comment]
It appears Thor 0.19.4 breaks `--no-color` and `help` somehow? @indirect is that something that I need to work around?

**Comment by segiddins** [comment]
@rafaelfranca can you confirm whether this breaking change was intended or not?

**Comment by rafaelfranca** [comment]
No breaking changes were intended. I can take a look later.

**Comment by segiddins** [comment]
👌🏻 I might also take a look, just wanted to check

**Comment by segiddins** [comment]
This will probably pass now

File: lib/bundler/cli.rb [hunk 1]
```diff
@@ -85,6 +85,9 @@ def help(cli = nil)
       end
     end
 
+    # Ensure `bundle help --no-color` is valid
+    all_commands["help"].disable_class_options = false
```
Comment (rafaelfranca) [inline lib/bundler/cli.rb]: Hmm. This is concerning. Seems like the change was to make sure help works without the required arguments, but in fact the change disable the option parsing. I'll try to fix it in a way that we don't broke option parser in the next version.

Comment (rafaelfranca) [inline lib/bundler/cli.rb]: See https://github.com/erikhuda/thor/pull/545

**Comment by segiddins** [comment]
@rafaelfranca I'll check out whether that PR fixes the bundler specs tomorrow

File: lib/bundler/cli.rb (hunk 1 above)
Comment (segiddins) [inline lib/bundler/cli.rb]: Confirmed that PR also keeps the Bundler specs green

**Comment by indirect** [comment]
cool, thanks!

@bundlerbot r+

File: lib/bundler/cli.rb (hunk 1 above)
Comment (rafaelfranca) [inline lib/bundler/cli.rb]: Great. I'll merge it and open a new PR removing this code.

Comment (segiddins) [inline lib/bundler/cli.rb]: It'd be great to get that fix (and also my io-console PR, if possible), into a release that we could pull in. Thanks so much for your work on this!

**Pull request merged**